src/document-templates/components/rentalConfirmationDocument.tsx

```diff
@@ -32,6 +39,12 @@ export const RentalConfirmationDocument: React.FC<Props> = ({ booking, globalSet
     const showPrices = booking.fixedPrice === null;
     const showPersonnelCosts = getTotalTimeEstimatesPrice(booking.timeEstimates).value !== 0 && showPrices;
 
+    const qrCodeImageSrc = getGlobalSetting(
+        'content.image.documentRentalAgreementQrCodeImage',
+        globalSettings,
+        '',
+    ).trim();
```

Comment:
Should we maybe trim in the `getGlobalSetting` function? Or is there a specific reason to only trim this one?

Reply:
I'm not a fan of "hidden" trims - there might be other setting where you want the whitespace preserved. Regarding this specific case, I believe the react-pdf had problems with strings with spaces before or after, so I added this to make it less prone to errors.